### PR TITLE
[Tankerkoenig] Updated README 

### PR DESCRIPTION
**Pull request by JueBag**
Updated README to suggest the use of the ephemeris action for setting the holiday switch.

Signed-off-by: Juergen Baginski opus42@gmx.de (github: JueBag)

### CODE REVIEW

**Comment by JueBag** [comment]
What?
I edited directly from the browser, signed with the usual signature, where did I miss signature?

File: bundles/org.openhab.binding.tankerkoenig/README.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ Note: If the API-key is banned by Tankerkönig, the reason has to be cleared wit
 
 The correct usage of opening times needs the information if the actual day is a holiday.
 The binding expects a switch item linked to the Webservice channel holiday.
-This switch can be set either manually (only suggested for testing!), by a rule &lsqb;openHAB1-addons rules&rsqb; or by the usage of the CALDAV binding with a calendar.
+This switch can be set either manually (only suggested for testing!) or by a rule which uses the [ephemeris action](<https://www.openhab.org/docs/configuration/actions.html#ephemeris>) to set that switch.
```
Comment (J-N-K) [inline bundles/org.openhab.binding.tankerkoenig/README.md]: Are you sure the <> around the link are needed?

Comment (JueBag) [reply bundles/org.openhab.binding.tankerkoenig/README.md]: Nope, it looked to me (on the preview) as if I needed them in order to show the link correctly. I could be wrong!

Comment (J-N-K) [reply bundles/org.openhab.binding.tankerkoenig/README.md]: I can override the DCO check, sign-off looks good to me.

Comment (JueBag) [reply bundles/org.openhab.binding.tankerkoenig/README.md]: Thanks for overriding, but I still do not understand this one problem.

**Pull request merged**